pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -124,16 +134,17 @@ type Values struct {
 
 // New creates a new instance of DeployWaiter for vpnshoot
 func New(
-	client client.Client,
+	client kubernetes.Interface,
```

Comment:
Do not pass the `kubernetes.Interface` if possible. You can pass `b.Shoot.KubernetesVersion` as part of the values instead (this is what other components do, I think).

Reply:
fixed